source/mir/las/sum.d

```diff
@@ -1782,18 +1782,56 @@ nothrow unittest
         tuple([2.0^^53+10.0, 1.0, 2.0^^-100], 2.0^^53+12.0),
         tuple([2.0^^53-4.0, 0.5, 2.0^^-54], 2.0^^53-3.0),
         tuple([M-2.0^^970, -1, M], 1.7976931348623157e+308),
-        tuple([double.max, double.max*2.^^-54], double.max),
-        tuple([double.max, double.max*2.^^-53], double.infinity),
```

Comment:
BTW, don't use this test for Pairwise summation  because they may gail on the future